Dz3/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Reflection.Metadata.Ecma335;
```

Comment:
What a heck is that??? O_o Why we need reflection... It is madness. :O I am joking, but still.

Reply:
Applications of reflection can include creating new objects, invoking methods, reading and modifying property and field values of objects, as well as exploring the program's structure during runtime to implement various algorithms and functionalities.